zenodo/modules/utils/cli.py

```diff
@@ -225,3 +228,21 @@ def list_files(recid):
             u'{idx:3}: "{key}", {checksum}, size:{size}'
             u''.format(idx=idx, key=key, checksum=f.checksum, size=f.size),
             fg='green'))
+
+
+@utils.command('sync_oai')
+@click.option('--eager', '-e', is_flag=True, default=False)
+@with_appcontext
+def sync_oai(eager):
+    """Update OAI IDs in the records."""
+    pids = PersistentIdentifier.query.filter(
+        PersistentIdentifier.pid_type == 'recid',
+        PersistentIdentifier.object_uuid is not None,
```

Comment:
+ PersistentIdentifier.object_type == 'rec',
- PersistentIdentifier.object_uuid is not None,